### ISSUE TITLE
Dashboard Widget

### ISSUE THREAD

**Issue by itsmerobliberty** [modmore/ClientConfig#133]
It's been asked before but without a real, non-Quickstart Buttons, answer. Is there any way to add the Configuration "Client View" directly to a Dashboard Widget for zero-click access? I made a quick mock up of what I'm looking to do.

Thank you so much for everything you do for the MODX community!

![screenshot-eternityhosting com-2017-06-20-10-09-38](https://user-images.githubusercontent.com/18542159/27337972-ec90e92c-55a1-11e7-8ed0-1e4f1ec2eb30.jpeg)

**Comment by Mark-H** [comment]
Happy to leave this open to see if there's more interest in it, but as it stands currently I don't have any immediate plans to add something like that. Appreciate the mockup tho, that makes it quite clear what you're looking for.

**Comment by c0mat0se** [comment]
I would absolutely love this feature! My clients could update certain variables easily right from the dashboard.

**Comment by itsmerobliberty** [comment]
Another benefit to this for me would be when the client user doesn't have access to the Extras menu (turned off in the ACL) they could still make those simple edits from their dashboard. 

If building out a full fledged widget doesn't sound interesting to you maybe it's possible to create a snippet with the Client Config page contents that we could put into our own widget?

As always, the Eternity crew is thankful for your work in the MODX community.

**Comment by crimsonpixel** [comment]
You can move the link to the Client Config from under the Extras menu and place on the top navigation and rename it to whatever you like via Lexicon.  I tend to do this for my clients and then add a Quick Link button to the area which I call "Site Settings" .

**Comment by greyskymedia** [comment]
+1

**Comment by itsmerobliberty** [comment]
@crimsonpixel Yes, that is a nice work around. But we would really love to see a widget, one less navigation item to deal with. We do use your navigation bar technique for our "View Site" and "Refresh MODX (Clear Cache)" buttons though, it works well.

**Comment by sebastian-marinescu** [comment]
+1
But I think I wouldn't want it for all settings, only for certain ones. 
Or even better would be some kind of Generator/Snippet-Call to custom-load the available settings in _a_ widget. That way we could have multiple widgets, and depending on the user-role or user-preference just show the settings that are necessary...

**Comment by itsmerobliberty** [comment]
@sebastian-marinescu Ah yes, that would be ideal if on the Admin grid you could specify which fields are "widget friendly". Or, like the Tagger extra, be able to create multiple "groups" that could contain a mix of fields and be assigned to certain user-groups... but that's icing on the cake, a simple widget editor would still make me happy :)

**Comment by Mark-H** [comment]
Well, that's definitely a lot more interest than I expected ;)

Would it make sense to have a widget where you choose which group to show? So instead of on the field level, you'd pick one of the groups already defined.

Also, with #123 coming along, any thoughts on how to deal with contexts in such a widget, if at all?

**Comment by itsmerobliberty** [comment]
Yes, I like the group idea.

We don't use a lot of additional contexts, so that feature isn't on our radar. I have to let someone else +1 that idea.

**Comment by itsmerobliberty** [comment]
Any updates on this?

**Comment by robliberty** [comment]
_bump_

**Comment by sebastian-marinescu** [comment]
> Would it make sense to have a widget where you choose which group to show? So instead of on the field level, you'd pick one of the groups already defined.

Yes, per-group choosing sounds good. 

> Also, with #123 coming along, any thoughts on how to deal with contexts in such a widget, if at all?

In multi-context-mode it would also need a little context-switcher in the top-right.
